### PR TITLE
Fix navbar dropdown positioning for all screen widths & better default avatar

### PR DESCRIPTION
**Pull request by dsernst**
From  https://github.com/okTurtles/group-income-simple/pull/280#issuecomment-310811145:
> But the horizontal positioning for the dropdown menu is off at different screen resolutions.

This PR fixes that issue, so that the navbar dropdown menu is correctly positioned for all screen widths.

### CODE REVIEW

**Comment by dsernst** [comment]
@taoeffect I believe this is ready now. 

Got the new default avatar in there too.

**Comment by taoeffect** [comment]
For figuring out if the failed builds are a return of the handleEvent/Logout conflict, we should have travis console output include output from the browser console, in a special color.

https://github.com/segmentio/nightmare#onconsole-functiontype--arguments-

**Comment by dsernst** [comment]
Yeah— I don't know why the builds are failing every time on Travis but I'm pretty sure it's not from my diff.

They pass on my local machine.

File: test/frontend.js [hunk 1]
```diff
@@ -143,7 +143,9 @@ describe('Frontend', function () {
         .insert('#LoginName', username)
         .insert('#LoginPassword', 'testtest')
         .click('#LoginButton')
-        .wait('#LogoutBtn')
+        .wait('#LoginResponse')
+        .wait(1000)
```
Comment (taoeffect) [inline test/frontend.js:147]: X this

**Comment by taoeffect** [comment]
Merged via #309

**Pull request closed without merging**